docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,384 @@
+`createIncidentJson`:: A stringified JSON with mustache variables that corresponds to *Create Case JSON*.
+`createIncidentResponseKey`:: A string from the response body of the create case method that corresponds to the *External Service Id*.
+`getIncidentUrl`:: A URL string with External Service Id mustache variable that corresponds to *Get Case URL*.
+`getIncidentResponseExternalTitleKey`:: A string from the response body of the get case method that corresponds to the *External Service Title*.
+`viewIncidentUrl`:: A URL string with either the External Service Id or External Service Title mustache variable that corresponds to *View Case URL*.
+`updateIncidentUrl`:: A URL string that corresponds to *Update Case URL*.
+`updateIncidentMethod`:: A string that corresponds to *Update Case Method*.
+`updateIncidentJson`:: A stringified JSON with mustache variables that corresponds to *Update Case JSON*.
+`createCommentUrl`:: A URL string that corresponds to *Create Comment URL*.
+`createCommentMethod`:: A string that corresponds to *Create Comment Method*.
+`createCommentJson`:: A stringified JSON with mustache variables that corresponds to *Create Comment JSON*.
+
+
+Secrets defines sensitive information for the connector type.
+
+`user`:: A string that corresponds to *User*. Required if `hasAuth` is set to `true`.
+`password`:: A string that corresponds to *Password*. Required if `hasAuth` is set to `true`.
+
+[float]
+[[define-cases-webhook-ui]]
+=== Define connector in Stack Management
+
+Define Webhook - Case Management connector properties.
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-connector.gif[Webhook - Case Management connector]
+
+Test Webhook - Case Management action parameters.
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-test.gif[Webhook - Case Management params test]
+
+[float]
+[[cases-webhook-action-configuration]]
+=== Action configuration
+
+Webhook - Case Management actions have the following configuration properties.
+
+Title:: A title for the issue, used for searching the contents of the knowledge base.
+Description:: The details about the incident.
+Labels:: The labels for the incident.
+Additional comments:: Additional information for the client, such as how to troubleshoot the issue.
+
+
+[float]
+[[cases-webhook-connector-full-example]]
+== Full Example with Third Party System
+
+The following is an example of using the Webhook - Case Management Connector with a real third party case management system, Jira. We will reference their https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/[API documentation] for creating an issue and use a demo instance of Jira with real working endpoints.
+
+_Note: We are using Jira as an example third party. However, if you wish to connect with Jira, using the <<jira-action-type, preconfigured Jira connector>> will simplify your setup._
+
+[float]
+====  Step 1 - Set up connector
```

Comment:
```suggestion
====  Set up connector
```

Reply:
Matching this with the text that's in the flyout so it's exactly the same.

Reply:
<img width="652" alt="Screen Shot 2022-08-08 at 8 10 40 AM" src="https://user-images.githubusercontent.com/6935300/183437997-1b45f24d-3a42-46dc-9350-fcc1f03fcf62.png">
i think we should leave the step numbers in, they are part of the flyout text